src/pages/main/main.tsx

```diff
@@ -0,0 +1,131 @@
+import PlaceCard from '../../components/place-card/place-card.tsx';
+
+type MainTotalOffersProps = {
+  totalOffers: number;
+}
+function MainPage({totalOffers}: MainTotalOffersProps): JSX.Element {
```

Comment:
Блоки кода отделяем пустой строкой
```suggestion
}

function MainPage({totalOffers}: MainTotalOffersProps): JSX.Element {
```